functions/resources/public/css/core.css

```diff
@@ -0,0 +1 @@
+html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video{margin:0;padding:0;border:0;font-size:100%;font:inherit;vertical-align:baseline}article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section{display:block}body{line-height:1}blockquote,q{quotes:none}blockquote:before,blockquote:after,q:before,q:after{content:'';content:none}table{border-collapse:collapse;border-spacing:0}*{box-sizing:border-box}ul,li{list-style-type:none}body,html{width:100%;height:100%}
```

Comment:
Do we actually want to commit this file if it's generated? A good way to avoid this would be including a .gitignore in the base of the project and splitting scss files into a directory separate from where we're generating them to.

Reply:
Yeah, absolutely, I'll add a gitignore, and in the future we'll just commit the .scss files.
I'll also add a sub folder structure so our sass files are organized (needed to do that at some point anyway)

Maybe something like:

/css         <---- outputed, non version controlled file gets dumped here
/css/scss <--- sass and partial files live here

Thoughts?